src/Tokenizer/Analyzer/Analysis/NamespaceUseAnalysis.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer\Analysis;
+
+/**
+ * @internal
+ */
+final class NamespaceUseAnalysis implements StartEndTokenAwareAnalysis
```

Comment:
just wondering, maybe it shall have only 2 properties - `namespaceAnalysis` and `isAliased` ?

Reply:
I would keep those 2 cases separate: use statements can also import functions and multiple classes at once. Currently that is not supported, but it would probably be easier not to mix this with the NamespaceAnalysis